docs/getting_started.md

```diff
@@ -3,7 +3,8 @@
 
 ## Prerequisites
 
-1. Python >= 3.7
+1. Python >= 3.7 and following dependency packages for fedora/centos
+   gcc, git, openssl-devel, python3-devel (or similar packages for ubuntu).
```

Comment:
Can we add another line in this list for these instead of adding to the python requirement?

Reply:
done